### PR TITLE
try to fix doc deployment

### PR DESCRIPTION
**Pull request by IAlibay**
I'll be honest I'm not 100% sure what's causing #91, I'm sure it's something super simple I'm not seeing.

### CODE REVIEW

**Comment by IAlibay** [comment]
don't merge as-is, I've just enabled PR docs building to allow us to diagnose the issue.

**Comment by orbeckst** [comment]
Isn't there a GH action for pages deployment that we could use instead?

**Comment by IAlibay** [comment]
> Isn't there a GH action for pages deployment that we could use instead?

https://github.com/marketplace/actions/github-pages-action or https://github.com/marketplace/actions/deploy-to-github-pages seem to do the trick.

They seem widely enough used if you want to go with either of them.

**Comment by orbeckst** [comment]
I'd be fine with either.

File: doc/source/conf.py [hunk 1]
```diff
@@ -61,8 +61,8 @@
 # General information about the project.
 project = u'GridDataFormats'
 authors = (u'Oliver Beckstein, Jan Domanski, Jesse Johnson, Max Linke, Tyler Luchko, '
-           u'Dominik Mierzejewski, Giacomo Fiorin, Lily Wang')
-copyright = u'2007-2020, ' + authors
+           u'Dominik Mierzejewski, Giacomo Fiorin, Lily Wang, Josh Vermaas, Irfan Alibay')
```
Comment (IAlibay) [inline doc/source/conf.py:64]: Apparently the author list doesn't get auto-populated. Does this need an issue?
Also am I good to bump up the copyright to 2021?

Comment (orbeckst) [reply doc/source/conf.py:64]: bump it up

you can raise an issue for auto-populating, especially as we already have the code somewhere (and this code was put under a special license IIRC...)

**Review by orbeckst — APPROVED** [review]
cool, thanks!!!

**Comment by IAlibay** [comment]
Let's see how this goes.. at least it builds now?

**Pull request merged**